source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -138,7 +144,9 @@ connection behavior:
        and decoding BSON binary values with subtype of 3.
 
    * - ``writeConcern()``
-     - Sets the :manual:`write concern </reference/write-concern/>`.
+     - | Sets the :manual:`write concern </reference/write-concern/>`.
+       |
+       | **Default**: ``"majority"``
```

Comment:
This is not the driver default.  The driver default is `WriteConcern#ACKNOWLEDGED` which defers to the server's configuration for default write concern (which on Atlas is indeed majority, iirc)